docs(list): web-first (#4823)
digdir/designsystem: apps/www/app/content/components/list/en/code.mdx

<Story story="Preview" defaultOpen/>

## Usage

You use either `<ul>` or `<ol>` with the class name `ds-list`.
`<li>` is used for each list item, and does not need a class name.

## CSS variables and data attributes

> Sizes are controlled with [data-size](/en/fundamentals/code/sizes) and colors with [data-color](/en/fundamentals/code/colors). The component will inherit the closest parent where these are set.

Reviewer: `list` is always neutral, so we might not need to mention color?

```suggestion
Sizes are controlled with [data-size](/en/fundamentals/code/sizes). The component will inherit the closest parent where this is set.
```
